packages/zoe/src/contractFacet/zcfZygote.js

```diff
@@ -232,7 +238,25 @@ export const makeZCFZygote = async (
     return evalContractBundle(bundle);
   };
   // evaluate the contract (either the first version, or an upgrade)
-  const { start, buildRootObject, meta = {} } = await evaluateContract();
+  const bundleResult = await evaluateContract();
+
+  const { buildRootObject, meta = {} } = bundleResult;
+
+  //#region backwards compatibility with prepare()
```

Comment:
> This ... sequesters the code in a region so it's easy to remove

I don't think the #region is serving the purpose you intend. In order to drop support for `prepare()`, different initializations for start and upgradeability would be needed.